portiaui/app/utils/selectors.js

```diff
@@ -879,6 +879,71 @@ export function makeItemsFromGroups(groups) {
     return items;
 }
 
+function createSelectorGenerators(structure, selectorMatcher, accumulator) {
```

Comment:
This is mutation `accumulator` and returns `generators`. Can we implement this without mutating a passed in parameter? If this is only used in `updateStructureSelectors` we don't need to mutate the `accumulator`, just construct it in the recursion by creating a helper function and then calling it in the invocation of `createSelectorGenerators`.